### PR TITLE
Py3 update

### PR DESCRIPTION
**Pull request by xcollantes**
Migrate Python2 to Python3 for pywifiphisher.py
This PR is a squashed continuation of (https://github.com/wifiphisher/wifiphisher/pull/1192).

### CODE REVIEW

**Comment by cclauss** [comment]
https://travis-ci.org/wifiphisher/wifiphisher/jobs/609849832#L728

File: wifiphisher/pywifiphisher.py [hunk 1]
```diff
@@ -451,7 +453,7 @@ def start(self):
                     "[{0}+{1}] Selecting {0}{2}{1} interface for creating the "
                     "rogue Access Point").format(G, W, ap_iface))
                 logger.info("Selecting {} interface for rogue Access Point"
-                            .format(ap_iface))
+                            .format(ap_iface)
```
Comment (cclauss) [inline wifiphisher/pywifiphisher.py]: Unbalanced ( )

**Comment by sophron** [comment]
I'm fine with merging this as long as it is in one commit.

**Comment by xcollantes** [comment]
@cclauss I hope I squashed that properly.  Looks like it's passing but not sure if I squashed too far.

**Comment by cclauss** [comment]
__ Files changed 0__ above is _not_ a good sign.

File: wifiphisher/pywifiphisher.py [hunk 2]
```diff
@@ -37,6 +32,8 @@
 import wifiphisher.common.recon as recon
 import wifiphisher.common.tui as tui
 import wifiphisher.common.victim as victim
+from wifiphisher.common.constants import *
```
Comment (cclauss) [inline wifiphisher/pywifiphisher.py]: https://www.asmeurer.com/removestar

Comment (xcollantes) [reply wifiphisher/pywifiphisher.py]: I can run *removestar*

File: wifiphisher/pywifiphisher.py [hunk 3]
```diff
@@ -241,7 +244,7 @@ def set_route_localnet():
     Popen(
         ['sysctl', '-w', 'net.ipv4.conf.all.route_localnet=1'],
         stdout=DN,
-        stderr=PIPE)
+        stderr=PIPE)<<<<<<< py3_update
```
Comment (cclauss) [inline wifiphisher/pywifiphisher.py]: Please revert this change

File: wifiphisher/pywifiphisher.py [hunk 4]
```diff
@@ -36,6 +36,16 @@
 from six.moves import range, input
 from wifiphisher.common.constants import *
 
+from six.moves import range, input
```
Comment (cclauss) [inline wifiphisher/pywifiphisher.py]: Line 36 or line 39 but not both.

**Comment by xcollantes** [comment]
@cclauss Hi Christian, I know it's not best practice and it's pretty sloppy but I might be posting commits here for a while until I get it right.  I'm still not used to this but I want to get it right.  

Do you mind if I ping you on here to let you know when I'm finished? Just so you're not having to come back to this for every commit.  Thanks.

File: wifiphisher/pywifiphisher.py [hunk 5]
```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python2
 # -*- coding: utf-8 -*-
 # pylint: skip-file
+from __future__ import absolute_import
+from __future__ import print_function
 import subprocess
```
Comment (xcollantes) [inline wifiphisher/pywifiphisher.py]: @cclauss I have a question: the pull request will only include the changes in green/red, right?  The rest of the code looks different from the wifiphisher/master base.

Comment (xcollantes) [reply wifiphisher/pywifiphisher.py]: @cclauss Another question: Should I rebase the merge of master to this branch as well?  Wouldn't my changes be hard to read?

**Comment by cclauss** [comment]
__Files changed 1__ means that only that file will be effected in this PR.
__This branch has no conflicts with the base branch__ means that file will merge cleanly.

The green/red lines are the only changes.

@ sophron can we merge this already?

**Comment by sophron** [comment]
Thank you @xcollantes.
